tests/functional/aws-node-sdk/test/object/unsupportedQueries.js

```diff
@@ -0,0 +1,39 @@
+import assert from 'assert';
+import makeRequest from '../../lib/utility/makeRequest';
+
+// Queries supported by AWS that we do not currently support.
+const unsupportedQueries = [
```

Comment:
You should define this list somewhere, utils, for example and import it everywhere you need it. This would avoid changing things in two place when we have to update.